src/Storage/RevisionableStorageInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity\Storage\RevisionableStorageInterface.
+ */
+
+namespace Drupal\entity\Storage;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+
+/**
+ * Defines some additional storage methods used for revision support.
+ */
+interface RevisionableStorageInterface extends EntityStorageInterface {
```

Comment:
as the method left is bound to having translation+revision, labeling it as "content" would make sense to me also.